src/components/Projects/index.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import Typography from 'material-ui/Typography';
+import Grid from 'material-ui/Grid';
+import Toolbar from 'material-ui/Toolbar';
+import Paper from 'material-ui/Paper';
+import Card, { CardContent } from 'material-ui/Card';
+import Link from 'react-router-dom/Link';
+import Loading from 'components/Utils/Loading';
+import styles from './style.module.css';
+import { observer, inject } from 'mobx-react';
+import { graphql } from 'react-apollo';
+import gql from 'graphql-tag';
+import BookmarkedIcon from '@material-ui/icons/Star';
+
+@graphql(gql`
+  query AllProjects($projectSearch: ProjectSearchInput){
+    projects(projectSearch: $projectSearch){
+      entries {
+        id
+        name
+        slug
+        bookmarked
+        environments {
+          id
+          name
+          key
+          __typename
+        }
+      }  
+      count
+    }
+  }
+`, {
+  options: (props) => ({
+    fetchPolicy: "network-only",
+    variables: {
+      projectSearch: {
+      },
+    },
+  })
+})
+
+@inject("store") @observer
+export default class Projects extends React.Component {
+  constructor(props){
+    super(props)
+    this.state = {}
+  }
+
+  render() {
+    const { loading, projects } = this.props.data;
+
+    if(loading || !projects){
```

Comment:
We may want to handle the condition `!loading && !projects` as a failure case. (as it should not happen? Or at the very least isn't the same thing as `loading`).

I don't think we currently handle that condition in master, so we don't need to change, but something to think about a bit to see how we want to handle things like that.

Reply:
```
    const { loading, project, error } = props.data;

    if(loading || !project || error){
        return null
    }
```
 work for you?